cmd/nerdctl/run_test.go

```diff
@@ -198,3 +199,43 @@ func TestRunEnv(t *testing.T) {
 		return nil
 	})
 }
+
+func TestRunWithLogOpt(t *testing.T) {
+	t.Parallel()
+	// docker use a different driver to support logging option
+	testutil.DockerIncompatible(t)
+	if runtime.GOOS == "windows" {
+		t.Skip("`logging options` are not yet implemented on Windows")
+	}
+	base := testutil.NewBase(t)
+	containerName := testutil.Identifier(t)
+
+	defer base.Cmd("rm", "-f", containerName).AssertOK()
+	base.Cmd("run", "-d", "--log-opt", "max-size=5k", "--log-opt", "max-files=2", "--name", containerName, testutil.CommonImage,
+		"sh", "-euxc", "du").AssertOK()
```

Comment:
Output of "du" doesn't seem enough to test the log rotation.

Reply:
`hexdump -C /dev/urandom  | head -nXXXX` is an easy way to generate random long text output